grantnav/frontend/context_processors.py

```diff
@@ -29,3 +29,7 @@ def main_css_cache_key(request):
 
 def debug_mode(request):
     return {"debug": settings.DEBUG}
+
+
+def insights_url(request):
```

Comment:
Do we need to add this processor to `TEMPLATES[0]["OPTIONS"]["context_processors"]` in `settings.py`? Or is this used some other way? I'm not really familiar with templates